openpype/hosts/maya/startup/userSetup.py

```diff
@@ -13,23 +12,40 @@
 
 print("Starting OpenPype usersetup...")
 
+settings = get_project_settings(os.environ['AVALON_PROJECT'])
+
+# Loading plugins explicitly.
+if settings["maya"]["explicit_plugins_loading"]["enabled"]:
+    def _explicit_load_plugins():
+        project_settings = get_project_settings(os.environ["AVALON_PROJECT"])
+        maya_settings = project_settings["maya"]
+        explicit_plugins_loading = maya_settings["explicit_plugins_loading"]
+        if explicit_plugins_loading["enabled"]:
+            for plugin in explicit_plugins_loading["plugins_to_load"]:
+                if plugin["enabled"]:
+                    print("Loading " + plugin["name"])
+                    try:
+                        cmds.loadPlugin(plugin["name"], quiet=True)
+                    except RuntimeError as e:
+                        print(e)
```

Comment:
This if statement is redundant right? Since we're already checking it above so it would never get here if it was disabled?
```suggestion
        for plugin in explicit_plugins_loading["plugins_to_load"]:
            if plugin["enabled"]:
                print("Loading plug-in: " + plugin["name"])
                try:
                    cmds.loadPlugin(plugin["name"], quiet=True)
                except RuntimeError as e:
                    print(e)
```

Also tweaked the print statement to mention it's loading a plug-in.